ash/src/extensions/khr/mod.rs

```diff
@@ -15,3 +15,25 @@ mod wayland_surface;
 mod win32_surface;
 mod xcb_surface;
 mod xlib_surface;
+
+pub fn memory_requirements2_name() -> &'static std::ffi::CStr {
```

Comment:
All those extension names are not necessary any more when https://github.com/MaikKlein/ash/pull/188 is merged.

Reply:
#188 has been merged. You can now do `vk::KhrGetMemoryRequirements2Fn::name()`.

Reply:
Done!